### ISSUE TITLE
&nbsp; support

### ISSUE THREAD

**Issue by thedavidmeister** [hoplon/hoplon#239]
HTML entities get escaped in strings, `(.fromCharCode js/String 160)` seems to be the best way to get spaces back in, would be nice if hoplon had a wrapper for such things

**Comment by flyboarder** [comment]
@thedavidmeister where are you using the escaped string? Can you give an example?

**Comment by thedavidmeister** [comment]
`(div "&nbsp;")` shows the string `&nbsp;` in the browser (rather than a space).

`(div (.fromCharCode js/String 160))` will create a div with a non breaking space in it

**Comment by flyboarder** [comment]
Are strings escaped with `(div :html “&nbsp;”)`?

**Comment by thedavidmeister** [comment]
no, i'm using `:html` to provide support for rendering the output of a WYSIWYG, so that's fine

**Comment by thedavidmeister** [comment]
`:html` is not a general solution though, for example I haven't yet looked into how the XSS story plays out when accepting raw HTML strings directly into the DOM
